app-emulation/img/img-0.5.7-r1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+EGO_PN="github.com/genuinetools/img"
+
+inherit golang-vcs-snapshot
+
+DESCRIPTION="Standalone daemon-less unprivileged Dockerfile and OCI container image builder"
+HOMEPAGE="https://github.com/genuinetools/img"
+SRC_URI="https://github.com/genuinetools/img/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64"
+LICENSE="MIT"
+SLOT="0"
+IUSE="seccomp"
+
+DEPEND="seccomp? ( sys-libs/libseccomp )"
+RDEPEND="${DEPEND}
+	app-emulation/runc"
+
+src_compile() {
+	GOPATH="${S}:$(get_golibdir_gopath)" \
+		GOCACHE="${T}/go-cache" \
+		IMG_DISABLE_EMBEDDED_RUNC=1 \
+		go build -v -work -x -tags "noembed $(usev seccomp)" \
```

Comment:
`usex`?

Reply:
https://devmanual.gentoo.org/eclass-reference/ebuild/index.html
```
usev <USE item>
    Like use, but also echoes USE item when use returns true. 
```

Reply:
Oh, sweet! Wasn't aware of this.